src/cli/linker.zig

```diff
@@ -160,16 +207,32 @@ pub fn link(allocator: Allocator, config: LinkConfig) LinkError!void {
         },
     }
 
+    // Add platform-provided files that come before object files
+    for (config.platform_files_pre) |platform_file| {
+        try args.append(platform_file);
+    }
+
     // Add object files
     for (config.object_files) |obj_file| {
         try args.append(obj_file);
     }
 
+    // Add platform-provided files that come after object files
+    for (config.platform_files_post) |platform_file| {
+        try args.append(platform_file);
+    }
+
     // Add any extra arguments
     for (config.extra_args) |extra_arg| {
         try args.append(extra_arg);
     }
 
+    // Debug: Print the linker command
+    std.log.debug("Linker command:", .{});
+    for (args.items) |arg| {
+        std.log.debug("  {s}", .{arg});
+    }
```

Comment:
This turned out to be very useful, so I left it here. Doesn't appear in release builds.